sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -510,11 +510,15 @@ rowFormat
     ;
 
 tableIdentifier
-    : (db=identifier '.')? table=identifier
+    : (db=identifierPart '.')? table=identifierPart
```

Comment:
Hi, @cloud-fan .
According to the PR title, the scope is **TABLE** name only, isn't it?
Please remove the change on `db=identifierPart` and the test cases.

[CREATE DATABASE](https://github.com/apache/spark/pull/19392/files#diff-8c1cb2af4aa1109e08481dae79052cc3R74)/DROP DATABASE/USE DATABASE still doesn't allow it.

Reply:
We should change the identifier of database to support this too.